istioctl/cmd/proxyconfig.go

```diff
@@ -576,7 +576,7 @@ func workloadConfigCmd() *cobra.Command {
   istioctl proxy-config workload <pod-name[.namespace]> --type HTTP --address 0.0.0.0 -o json
 
   # Retrieve workload summary without using Kubernetes API
```

Comment:
this isn't really "not using Kubernetes API"

Reply:
updated!